File: sources/WCSF2010/Trunk/Source/WebClientFactoryGP/WebClientFactoryPackage/Templates/Projects/Module.CS/Views/DefaultViewPresenter.cs
//===============================================================================
// Microsoft patterns & practices
// Web Client Software Factory 2010
//===============================================================================
// Copyright (c) Microsoft Corporation.  All rights reserved.
// THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY
// OF ANY KIND, EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT
// LIMITED TO THE IMPLIED WARRANTIES OF MERCHANTABILITY AND
// FITNESS FOR A PARTICULAR PURPOSE.
//===============================================================================
// The example companies, organizations, products, domain names,
// e-mail addresses, logos, people, places, and events depicted
// herein are fictitious.  No association with any real company,
// organization, product, domain name, email address, logo, person,
// places, or events is intended or should be inferred.
//===============================================================================
using System;
using System.Collections.Generic;
using System.Text;
using Microsoft.Practices.ObjectBuilder;
using Microsoft.Practices.CompositeWeb;

namespace $ModuleNamespace$.Views
{
    public class DefaultViewPresenter : Presenter<IDefaultView>
    {
        private I$ModuleName$Controller _controller;

        public DefaultViewPresenter([CreateNew] I$ModuleName$Controller controller)
        {
            this._controller = controller;
        }        
    }
}

File: sources/WCSF2010/Trunk/Source/WebClientFactoryGP/WebClientFactoryPackage/bin/Debug/Templates/Projects/Module.VB/Views/DefaultViewPresenter.vb
Imports System
Imports System.Collections.Generic
Imports System.Text
Imports Microsoft.Practices.ObjectBuilder
Imports Microsoft.Practices.CompositeWeb

Namespace Views

    Public Class DefaultViewPresenter
        Inherits Presenter(Of IDefaultView)

        Private _controller As I$ModuleName$Controller

        Public Sub New(<CreateNew()> ByVal moduleController As I$ModuleName$Controller)
            _controller = moduleController
        End Sub

    End Class

End Namespace
